packages/server/src/automations/tests/utilities/AutomationTestBuilder.ts

```diff
@@ -43,69 +46,55 @@ type TriggerOutputs =
   | CronTriggerOutputs
   | undefined
 
-class AutomationBuilder {
-  private automationConfig: Automation = {
-    name: "",
-    definition: {
-      steps: [],
-      trigger: {} as AutomationTrigger,
-    },
-    type: "automation",
-    appId: setup.getConfig().getAppId(),
-  }
-  private config: TestConfiguration = setup.getConfig()
-  private triggerOutputs: TriggerOutputs
-  private triggerSet: boolean = false
-
-  constructor(options: { name?: string } = {}) {
-    this.automationConfig.name = options.name || `Test Automation ${uuidv4()}`
+type BranchConfig = {
+  [key: string]: {
+    steps: (stepBuilder: StepBuilder) => StepBuilder
```

Comment:
```suggestion
    steps: (stepBuilder: StepBuilder) => void
```

Because you don't use the return value of this function, I'd make it void.